src/modules/classList.js

```diff
@@ -0,0 +1,17 @@
+export default class ScoresList {
+  async addscore(names, scorevalue) {
+    
+    fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/U048O1pn604XSuQgI6WO/scores', {
+      method: 'POST',
+      body: JSON.stringify({
+        user: names,
+        score: scorevalue,
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    });
+    await ((response) => response.json());
+    await ((error) => error('Error:', error));
+  }
```

Comment:
- [ ] Kindly refactor this function to an arrow function as the previous reviewer requested you in other functions as well.

Reply:
Thank you